beacon-chain/blockchain/BUILD.bazel

```diff
@@ -126,11 +125,11 @@ go_test(
         "@com_github_prysmaticlabs_ethereumapis//eth/v1alpha1:go_default_library",
         "@com_github_sirupsen_logrus//:go_default_library",
         "@com_github_sirupsen_logrus//hooks/test:go_default_library",
-        "@com_github_gogo_protobuf//types:go_default_library",
         "@in_gopkg_d4l3k_messagediff_v1//:go_default_library",
         "@org_golang_x_net//context:go_default_library",
-        "@org_golang_google_grpc//codes:go_default_library",
-        "@org_golang_google_grpc//status:go_default_library",
+        "//beacon-chain/core/feed/block:go_default_library",
```

Comment:
I think you don't need to import this because it's included already as `embed = [":go_default_library"],` in your `go_test`